src/msg/simple/Pipe.cc

```diff
@@ -427,9 +427,6 @@ int Pipe::accept()
       goto fail_unlocked;
     }
 
-    // sanitize features
-    connect.features = connect.features;
-
```

Comment:
Did you check the history on these? I don't recognize them but the warning made me concerned they are meant to do something (say, sanitize the feature list into something we understand?) via C++ magic that has failed.